src/main/kotlin/com/linchpino/core/service/AccountService.kt

```diff
@@ -35,6 +40,21 @@ class AccountService(
     fun findMentorsWithClosestTimeSlotsBy(date: ZonedDateTime, interviewTypeId: Long): List<MentorWithClosestTimeSlot> {
         val from = date.withZoneSameInstant(ZoneOffset.UTC)
         val to = from.plusHours(24)
-        return repository.closestMentorTimeSlots(from,to, interviewTypeId)
+        return repository.closestMentorTimeSlots(from, to, interviewTypeId)
+    }
+
+    fun activeJobSeekerAccount(request: ActivateJobSeekerAccountRequest): AccountSummary {
+        val account = repository.findByExternalId(request.externalId, AccountTypeEnum.JOB_SEEKER)
+            ?: throw RuntimeException("account not found")
+        if (account.status == AccountStatusEnum.ACTIVATED)
+            throw RuntimeException("account is already activated")
+        val updatedAccount = account.apply {
+            firstName = request.firstName
+            lastName = request.lastName
+            password = request.password
```

Comment:
The password is not encrypted. I have defined another issue to refactor the add account to point all places to a generic point. I mean only one point should call accountRepository.save

Reply:
I updated the pr with encoded password for now.